### PR TITLE
Sprites for drinks without sprite

### PR DESCRIPTION
**Pull request by RumiTiger**
## About the PR
I added sprites in a metamorph glass for drinks such as:
• Coffee
• Cola
• Dr Gibb
• Green tea
• Ice
• Iced green tea
• Lemon juice
• Moonshine
• Orange juice
• Space mountain wind
• Space-up
• Sugar
• Tea
• Tequilla
• Tonic
• Watermelon juice

## Why / Balance
Why not?

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
photo_2024-03-11_19-02-25

• Coffee
photo_2024-03-11_19-13-21

• Cola
image

• Dr Gibb
photo_2024-03-11_19-13-20

• Green tea
photo_2024-03-11_19-13-21 (3)

• Ice
photo_2024-03-11_20-11-06

• Iced green tea
photo_2024-03-11_19-13-20 (2)

• Lemon juice
photo_2024-03-11_19-13-20 (3)

• Moonshine
image

• Orange juice
photo_2024-03-11_19-13-21 (5)

• Space mountain wind
photo_2024-03-11_19-13-20 (4)

• Space-up
photo_2024-03-11_19-13-21 (6)

• Sugar
photo_2024-03-12_14-17-22

• Tea
image

• Tequilla
photo_2024-03-11_19-13-21 (8)

• Tonic
photo_2024-03-11_19-13-21 (9)

• Watermelon juice
photo_2024-03-12_12-18-29

**Changelog**
🆑
• add: Added sprites for some drinks without their own sprite and changed the sprites of some drinks

### CODE REVIEW

**Comment by RumiTiger** [comment]
> moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.

Do I need to make the watermelon juice sprite brighter?

**Comment by UbaserB** [comment]
> > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> 
> Do I need to make the watermelon juice sprite brighter?

more vibrant and darker, probably? Maybe even add a gradient

**Comment by RumiTiger** [comment]
> > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > 
> > 
> > Do I need to make the watermelon juice sprite brighter?
> 
> more vibrant and darker, probably? Maybe even add a gradient

Like this?

**Comment by RumiTiger** [comment]
> > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > 
> > 
> > Do I need to make the watermelon juice sprite brighter?
> 
> more vibrant and darker, probably? Maybe even add a gradient

Like this?

**Comment by UbaserB** [comment]
> > > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > > 
> > > 
> > > Do I need to make the watermelon juice sprite brighter?
> > 
> > 
> > more vibrant and darker, probably? Maybe even add a gradient
> 
> Like this? photo_2024-03-12_12-18-29

it is better yes

**Comment by RumiTiger** [comment]
> moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.

What's wrong with moonshine?

**Comment by UbaserB** [comment]
> > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> 
> What's wrong with moonshine?

its hard to tell what its supposed to be and just doesnt make much sense, it would be better to replace it with something like this:

image

File: Resources/Textures/Objects/Consumable/Drinks/tequillaglass.rsi/meta.json [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
+      "name": "icon_empty"
+    },
+    {
+      "name": "fill-1"
+    },
+    {
+      "name": "fill-2"
+    },
+    {
+      "name": "fill-3"
```
Comment (UbaserB) [inline Resources/Textures/Objects/Consumable/Drinks/tequillaglass.rsi/meta.json:23]: Missing fill-4

**Comment by RumiTiger** [comment]
> ts hard to tell what its supposed to be and just doesnt make much sense, it would be better to replace it with something like this:

I tried to draw moonshine in a cut glass
image

**Comment by UbaserB** [comment]
> > ts hard to tell what its supposed to be and just doesnt make much sense, it would be better to replace it with something like this:
> 
> I tried to draw moonshine in a cut glass

its hard to do with this low number of pixels, might as well pick something else

**Comment by RumiTiger** [comment]
> > > ts hard to tell what its supposed to be and just doesnt make much sense, it would be better to replace it with something like this:
> > 
> > 
> > I tried to draw moonshine in a cut glass
> 
> its hard to do with this low number of pixels, might as well pick something else

Okay, I'll think of something

**Comment by RumiTiger** [comment]
> moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.

Maybe make a glass of sugar like this?
image

**Comment by UbaserB** [comment]
> > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> 
> Maybe make a glass of sugar like this?

that does not look like you can drink from it.

**Comment by RumiTiger** [comment]
> > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > 
> > 
> > Maybe make a glass of sugar like this?
> 
> that does not look like you can drink from it safely.

But it is ideal for table decoration

**Comment by UbaserB** [comment]
> > > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > > 
> > > 
> > > Maybe make a glass of sugar like this?
> > 
> > 
> > that does not look like you can drink from it safely.
> 
> But it is ideal for table decoration

I know some characters, like moths, who enjoy drinking sugar. Also, its liquid; not powder.

**Comment by RumiTiger** [comment]
> > > > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > > > 
> > > > 
> > > > Maybe make a glass of sugar like this?
> > > 
> > > 
> > > that does not look like you can drink from it safely.
> > 
> > 
> > But it is ideal for table decoration
> 
> I know some characters, like moths, who enjoy drinking sugar. Also, its liquid; not powder.

Is sugar a liquid? I thought it was a game convention.

**Comment by UbaserB** [comment]
> > > > > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > > > > 
> > > > > 
> > > > > Maybe make a glass of sugar like this?
> > > > 
> > > > 
> > > > that does not look like you can drink from it safely.
> > > 
> > > 
> > > But it is ideal for table decoration
> > 
> > 
> > I know some characters, like moths, who enjoy drinking sugar. Also, its liquid; not powder.
> 
> Is sugar a liquid? I thought it was a game convention.

some sugars are yes

**Comment by Tayrtahn** [comment]
Bless you for including fill levels for all of these!

**Comment by potato1234x** [comment]
the space cola one looks a bit wrong, make it just a brown liquid with bubbles like tg

**Comment by RumiTiger** [comment]
> the space cola one looks a bit wrong, make it just a brown liquid with bubbles like tg

Maybe this?
image

**Comment by UbaserB** [comment]
> > the space cola one looks a bit wrong, make it just a brown liquid with bubbles like tg
> 
> Maybe this? image

looks cherry-flavoured

**Comment by UbaserB** [comment]
better i think

**Comment by RumiTiger** [comment]
> > > moonshine and sugar sprites look a bit odd (moonshine doesnt make sense and sugar shouldnt be a coconut), and the colour for the watermelon drink is too washed out.
> > 
> > 
> > What's wrong with moonshine?
> 
> its hard to tell what its supposed to be and just doesnt make much sense, it would be better to replace it with something like this:

image

**Comment by pigeonpeas** [comment]
the tea sprite is a lil odd, but i like the rest. Maybe add a handle to the teacup?

**Comment by RumiTiger** [comment]
> the tea sprite is a lil odd, but i like the rest. Maybe add a handle to the teacup?

The problem is that there are already mugs in the game, but I can draw a tea sprite similar to green tea

**Comment by RumiTiger** [comment]
> the tea sprite is a lil odd, but i like the rest. Maybe add a handle to the teacup?

image
Maybe this?

**Comment by RumiTiger** [comment]
> I just noticed that you're using colored sprites for the fill levels. That's fine for the more visually complicated fills, but it would be great to use white or greyscale for the ones that are more or less a solid color. Set metamorphicChangeColor to true, and their sprite colors will be multiplied by the reagent color. Pure white will show up as the reagent color, and grays will show up as darker shades of it (useful for shading). You can use the alpha channel too, which works well if you want a portion to look like it's tinted by the glass in front of it (like when the fill passes "under" the front of the glass' rim)
> 
> This makes it easier to change the reagent/sprite color in the future, and also lets the colors blend in a mixed solution. So if someone puts a little lemon juice into a glass of tonic water, it'll take on a slightly yellowish hue, which I find pretty cool.
> 
> I wouldn't block merging this PR over it, but it would be a nice bonus you could add if you have time.

Maybe I'll do it later

**Comment by metalgearsloth** [comment]
Undraft it when it's ready for a review and you've tested it.

**Comment by BRINGit34** [comment]
This is just me spitballing here but shouldn't moonshine use a mason jar? That seems to fit the aesthetic of moonshine more

**Pull request closed without merging**